linux_os/guide/system/software/integrity/crypto/openssl_use_strong_entropy/rule.yml

```diff
@@ -0,0 +1,34 @@
+documentation_complete: true
+
+# TODO: The plan is not to need this for RHEL>=8.4
+prodtype: rhel8
+
+title: 'OpenSSL uses strong entropy source'
+
+description: |-
+    To set up an <tt>openssl</tt> wrapper that adds a <tt>-rand /dev/random</tt> option to the <tt>openssl</tt> invocation,
+    save the following shell snippet to the <tt>/etc/profile.d/openssl-rand.sh</tt>:
+    <pre>
+    {{{ openssl_strong_entropy_config_file() | indent(4) }}}
+    </pre>
+
+rationale: |-
+    This rule ensures that <tt>openssl</tt> always uses SP800-90A compliant random number generator.
+
+severity: medium
+identifiers:
+    cce@rhel8: 82721-2
+
+references:
+    ospp: FIA_AFL.1
+
+ocil: |-
+    To determine whether the <tt>openssl</tt> wrapper is configured correctly,
+    make sure that the <tt>/etc/profile.d/openssl-rand.sh</tt> file contains contents
+    that are included in the rule's description.
```

Comment:
Similar to comment above, the goals is to make sure OpenSSL always use the compliant source of entropy. And the wrapper is a way to achieve this.